src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -428,32 +435,37 @@ function ChildReconciler(shouldClone, shouldTrackSideEffects) {
           const created = createFiberFromElement(newChild, priority);
           created.ref = coerceRef(null, newChild);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_COROUTINE_TYPE: {
           const created = createFiberFromCoroutine(newChild, priority);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_YIELD_TYPE: {
           const created = createFiberFromYield(newChild, priority);
           created.type = newChild.value;
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_PORTAL_TYPE: {
           const created = createFiberFromPortal(newChild, priority);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
       }
 
       if (isArray(newChild) || getIteratorFn(newChild)) {
         const created = createFiberFromFragment(newChild, priority);
         created.return = returnFiber;
+        created.contextTag = returnFiber.contextTag;
```

Comment:
Now that I'm seeing this whole thing, I'm noticing we always do these two things:
```
created.return = returnFiber;
created.contextTag = returnFiber.contextTag;
```

Makes me think of putting these two things into a reusable method, 'updateFiber' or something, which takes the 'created' and 'returnFiber'. I might think differently once I finish reading this code more thoroughly though. And it seems like we err on the side of less abstraction, so maybe the repetition is ok.